tours-cernbox/new_features.json

```diff
@@ -0,0 +1,73 @@
+        },
+        {
+          "title": "2.  Single File Sharing",
+          "text": "<div><p>Now, it's possible to share not only folders but also single files in both read and write modes!\r\n        </p><img class=\"guide-img\" src=\"https://new-cernbox-guide.docs.cern.ch/assets/screenshots/sharing.png\"></img></div>",
+          "buttons": ["back", "next"],
+          "learnMoreLink": "https://new-cernbox-guide.docs.cern.ch/sharing/#internal-shares"
+        },
+        {
+          "title": "3. Improved App Ecosystem",
+          "text": "<div><p>The new CERNBox has diverse application/file extensions integrations: Microsoft Office 365, Collabora, CodiMD, Draw.io, text editor, PDF viewer, IFC viewer, Jupyter viewer, Root viewer, SWAN, Media viewer.\r\n        </p><img class=\"guide-img\" src=\"https://new-cernbox-guide.docs.cern.ch/assets/screenshots/apps-context.png\"></img></div>",
+          "buttons": ["back", "next"],
+          "learnMoreLink": "https://new-cernbox-guide.docs.cern.ch/apps/"
+        },
+        {
+          "title": "4. Improved Projects Integration",
+          "text": "<div><p>Now it's also possible to see and recover files of projects by right clicking on a project and then on \"Open trahsbin\"\r\n        </p><img class=\"guide-img\" src=\"https://new-cernbox-guide.docs.cern.ch/assets/screenshots/projects-trashbin.png\"></img></div>",
+          "buttons": ["back", "next"],
+          "learnMoreLink": "https://new-cernbox-guide.docs.cern.ch/projects/"
+        },
+        {
+          "title": "Lightweight accounts",
+          "text": "<div><p>The new CERNBox uses the new CERN SSO and it now allows signing in with other accounts, like social or lightweight.\r\n        </p><p>Unlike the normal CERN accounts, these do not have storage associated with them. They can, however, collaborate on shared resources or on Projects.</p><img class=\"guide-img\" src=\"https://new-cernbox-guide.docs.cern.ch/assets/screenshots/lightweight.png\"></img></div>",
+          "buttons": ["back", "next"],
+          "learnMoreLink": "https://new-cernbox-guide.docs.cern.ch/access/#accounts-supported"
+        },
+        {
+          "title": "Coming soon",
+          "text": "<div><p>Other features will be coming in the coming months:</p>\r\n        <oc-list><li>Integrations with other services (i.e Indico)</li>\r\n        <li>Improved search</li>\r\n        <li>Auditing and reporting/notifications</li>\r\n        <li>Backup restore UI</li></oc-list>\r\n        <p> Stay tuned!</p></div>",
+          "buttons": ["back", "next"]
+        }
+      ]
+    }
+  },
+  "confirmCancel": false,
+  "confirmCancelMessage": "",
+  "classPrefix": "",
+  "defaultStepOptions": {
+    "cancelIcon": {
+      "enabled": true
+    },
+    "classes": "a",
+    "scrollTo": {
+      "behavior": "smooth",
+      "block": "center"
+    }
+  },
+  "exitOnEsc": true,
+  "allowedLocations": [],
```

Comment:
Can we configure the allowed locations instead? I would prefer to see this only in the files listing, instead of any external app (which is the case)